contributing/chestertons-fence.md

```diff
@@ -84,3 +84,4 @@ Two things:
 
 1. [This article in Second Order Thinking](https://fs.blog/2020/03/chestertons-fence/) is a good read. The quote "Do not remove a fence until you know why it was put up in the first place" is from this post.
 2. Your idea really might be new and great. Don't be scared or embarrassed to suggest it!
+3. G.K. Chesteron's, _The Thing_ is available [https://archive.org/details/G.K.ChestertonTheThing](https://archive.org/details/G.K.ChestertonTheThing).
```

Comment:
Two comments: 
1. Could you switch points two and three around, so that the "Don't be scared" is the last point still.
2. We need to update "Two things:" above this to say "Three things:"

Reply:
(Thanks btw!)

Reply:
Attempting to use "account numbering" in markdown, let me know if that works.

Reply:
I guess it isn't really a footnote, I spent some time trying to find the footnote indicators in the body, and then realized this is just an ordered list.

A bullet list might be better, stylistically.  (Just observing aloud.)

Reply:
Merging. We can always improve in further PRs! Thanks :)

Reply:
I could have changed to to "10 things" and it would be  correct, we would just mentally adjust the base.

Reply:
Ha! :)